libs/core/src/scss/base/_item-typography.scss

```diff
@@ -1,4 +1,4 @@
 @use 'sass:list';
 
-$slotted-text-elements-base: h1, h2, h3, h4, h5, h6, data;
```

Comment:
Removing `data` from this list results in some general changes to the color, display and margin properties of the element when used in item (see [_item.utils.scss](https://github.com/kirbydesign/designsystem/blob/develop/libs/designsystem/item/src/_item.utils.scss#L23-L31) and [_typography.scss](https://github.com/kirbydesign/designsystem/blob/develop/libs/core/src/scss/base/_typography.scss#L35-L36)). 

The unintended visual changes are also visible in chromatic tests.

So I think another approach is needed where `data` is excluded from the text overflow styles specifically.